Sources/Debugging/NSError+Debuggable.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+extension NSError: Debuggable {
+    
+    /// See `Debuggable`
+    public var identifier: String {
+        return "\(self.code)"
+    }
+    
+    /// See `Debuggable`
+    public var reason: String {
+        return self.debugDescription
+    }
+}
```

Comment:
What if we set possibleCauses to self.localizedFailureReason and suggestedFixes to self.localizedRecoverySuggestion? I'm not sure if they will ever be helpful, but it might be.

Reply:
Sounds good! Added :)